backend/people/migrations/0019_fill_more_seminars.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
```

Comment:
The seminar model is in the events app. So, this migration must be in the events app.

Reply:
Done